CONTRIBUTING.md

```diff
@@ -51,6 +51,65 @@ implementation of the issue, ask them in the issue instead of the PR.
 
 The following instructions are for developers and contributors to cuDF OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuDF from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
 
+### Code Formatting
+
+#### Python
+
+cuDF uses [Black](https://black.readthedocs.io/en/stable/),
+[isort](https://readthedocs.org/projects/isort/), and
+[flake8](http://flake8.pycqa.org/en/latest/) to ensure a consistent code format
+throughout the project. `Black`, `isort`, and `flake8` can be installed with
+`conda` or `pip`:
+
+```bash
+conda install black isort flake8
+```
+
+```bash
+pip install black isort flake8
```

Comment:
Since we _require_ conda and don't support pip at all, wouldn't it make sense to remove instructions on installing these with pip?

Reply:
I had that thought, but if users want to install some of the isolated parts of their toolchain with pip, then that's their prerogative. I figure give people the choice for things like this.

Reply:
Alright, it's not a big issue, was really just wondering if we would like to be consistent in that sense.